evergreen/prepare-ocsp.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+set -o xtrace   # Write all commands first to stderr
+set -o errexit  # Exit the script with an error if any of the commands fail
+
+cd ${DRIVERS_TOOLS}/.evergreen/ocsp
+
+echo "Preparing Python env for OCSP tests"
+
+if [ "$OSTYPE" = "cygwin" ]; then
+  /cygdrive/c/python/python38/python.exe -m venv ./venv
+  ./venv/Scripts/pip3 install -r ${DRIVERS_TOOLS}/.evergreen/ocsp/mock-ocsp-responder-requirements.txt
+else
+  which python
```

Comment:
The OCSP tests on master fail here as well, this is just a start to get moving forward when that ticket lands.

Reply:
Added comment.